src/app/events/SortEvent.ts

```diff
@@ -0,0 +1,4 @@
+export interface SortEvent {
+    direction: string;
+    themeId: string;
```

Comment:
Maybe we should use Message internally instead of Theme

Reply:
Where do you think this would apply? Every bit of code (including html) and then only the displayed title would show theme to the user? I have used it throughout so it would be a non-trivial amount of time to make this change